vale/styles/vocab.txt

```diff
@@ -36,11 +37,13 @@ JWT
 JWTs
 Kuzco
 Memcache
+MacOS
```

Comment:
Please change this to 'macOS' - it doesn't match our (relatively clearer) guidance on brand capitalisation - see [note in this section of the writing style guide](https://github.com/buildkite/docs/blob/main/styleguides/writing-style.md#product-names-and-product-features).